pkg/porter/create.go

```diff
@@ -1,43 +1,73 @@
 package porter
 
 import (
+	"errors"
 	"fmt"
 	"os"
 	"path/filepath"
+	"strings"
 
 	"get.porter.sh/porter/pkg"
 	"get.porter.sh/porter/pkg/config"
 )
 
-func (p *Porter) Create() error {
-	fmt.Fprintln(p.Out, "creating porter configuration in the current directory")
+// Create creates a new bundle configuration with the specified bundleName. A directory with the given bundleName will be created if it does not already exist.
+// If bundleName is the empty string, the configuration will be created in the current directory, and the name will be "porter-hello".
+func (p *Porter) Create(bundleName string) error {
+	// Normalize the bundleName by removing trailing slashes
+	bundleName = strings.TrimSuffix(bundleName, "/")
 
-	err := p.CopyTemplate(p.Templates.GetManifest, config.Name)
+	// If given a bundleName, create directory if it doesn't exist
+	if bundleName != "" {
+		_, err := os.Stat(bundleName)
+		if err != nil && errors.Is(err, os.ErrNotExist) {
+			err = os.Mkdir(bundleName, os.ModePerm)
+			if err != nil {
+				return fmt.Errorf("failed to create directory for bundle: %w", err)
+			}
+		}
+	}
+
+	var err error
+	if bundleName == "" {
```

Comment:
Your logic is repeating itself if it falls through the bundleName case being empty. Returning early with nil would make this check implicit if it isn’t filled out here. 

The logic is the same in both cases it’s just where it gets executed. This can make it easier to understand and less things to write.

Reply:
hopefully this is addressed now that it's 2 different functions